desktop/core/src/desktop/js/apps/storageBrowser/StorageBrowserPage/StorageBrowserTable/StorageBrowserTable.tsx

```diff
@@ -61,12 +71,39 @@ const StorageBrowserTable: React.FC<StorageBrowserTableProps> = ({
 
   const { t } = i18nReact.useTranslation();
 
+  const onColumnTitleClicked = (columnClicked: string) => {
+    if (columnClicked === sortByColumn) {
+      onSortByDescendingChange(!sortByDescending);
+    } else {
+      onSortByColumnChange(columnClicked);
+      if (sortByDescending) {
+        onSortByDescendingChange(false);
+      }
+    }
+  };
+
   const getColumns = file => {
     const columns: ColumnProps<unknown>[] = [];
     for (const [key] of Object.entries(file)) {
       const column: ColumnProps<unknown> = {
         dataIndex: `${key}`,
-        title: t(`${key}`.charAt(0).toUpperCase() + `${key}`.slice(1)),
+        title: (
+          <div
+            className="hue-storage-browser__table-column-title"
+            onClick={() => onColumnTitleClicked(key)}
+          >
+            <div>{t(`${key}`.charAt(0).toUpperCase() + `${key}`.slice(1))}</div>
```

Comment:
Are you making first letter uppercase in order to match on the i18n keys or is it because you want to display the string with first letter uppercase? If the later you should use CSS.

Reply:
I was trying to display it that way. I'll switch it to css